app/javascript/sdk/bubbleHelpers.js

```diff
@@ -65,11 +65,6 @@ export const createBubbleHolder = hideMessageBubble => {
   body.appendChild(bubbleHolder);
 };
 
-export const createNotificationBubble = () => {
```

Comment:
@pranavrajs, Could you please verify? These changes were added by this [PR](https://github.com/chatwoot/chatwoot/pull/653/files#diff-ca628bbfec83843b4e335c0cf19bddb91335b30cdcb428d0fc6a5930bcc3237aR29)

Reply:
I just moved it, didn't write it in the first place. Let me test it out.